src/Simplex/Chat.hs

```diff
@@ -942,13 +942,14 @@ processAgentMessage toView user@User {userId, profile} agentConnId agentMessage
           group <- withStore $ \st -> getGroup st user gName
           case find (sameMemberId memId) $ members group of
             Nothing -> messageError "x.grp.mem.inv error: referenced member does not exists"
-            Just reMember -> do
-              intro <- withStore $ \st -> saveIntroInvitation st reMember m introInv
-              case activeConn (reMember :: GroupMember) of
-                Nothing -> pure () -- this is not an error, introduction will be forwarded once the member is connected
+            Just reMember@GroupMember {groupMemberId = reMemberId} -> do
+              GroupMemberIntro {introId} <- withStore $ \st -> saveIntroInvitation st reMember m introInv
+              (msgId, msgBody) <- createSndMessage $ XGrpMemFwd (memberInfo m) introInv
+              case memberConn reMember of
+                Nothing -> withStore $ \st -> createPendingGroupMessage st reMemberId msgId (Just introId)
```

Comment:
I don't think it is right... You need only current members here, not all members (that would include those that are invited to the group by the current user but did not accept yet).